receiver/splunksearchapireceiver/client.go

```diff
@@ -60,6 +61,10 @@ func newSplunkSearchAPIClient(ctx context.Context, settings component.TelemetryS
 func (c defaultSplunkSearchAPIClient) CreateSearchJob(search string) (CreateJobResponse, error) {
 	endpoint := fmt.Sprintf("%s/services/search/jobs", c.endpoint)
 
+	if !strings.Contains(search, "starttime=") || !strings.Contains(search, "endtime=") || !strings.Contains(search, "timeformat=") {
```

Comment:
Might be overkill but in splunk, are these case-insensitive? I.e. I believe you  can do `Starttime="<whatever>` and its still valid

For the sake of correctness I'm curious if we need to make these error checks non case-sensitive? 

I know that we're inputting these parameters so its not too big of a deal, so maybe a comment would suffice in this situation

Reply:
Good catch, I didn't know that about Splunk.